beacon/weakcoin/weak_coin_test.go

```diff
@@ -1,4 +1,4 @@
-package weakcoin_test
+package weakcoin
```

Comment:
why? using _test package is a better practice

Reply:
changed back. i am aware this is better practice, but for the most part i find impractical and not consistent with the rest of the codebase.